tuf_conformance/test_updater_delegation_graphs.py

```diff
@@ -0,0 +1,127 @@
+from dataclasses import dataclass, field
+from typing import Any
+
+import pytest
+from tuf.api.metadata import (
+    TOP_LEVEL_ROLE_NAMES,
+)
+
+from tuf_conformance.client_runner import ClientRunner
+from tuf_conformance.simulator_server import SimulatorServer
+
+# DataSet is only here so type hints can be used.
+DataSet = dict[str, Any]
+
+
+@dataclass
+class DelegationTester:
+    delegator: str
+    rolename: str
+    keyids: list[str] = field(default_factory=list)
+    threshold: int = 1
+    terminating: bool = False
+    paths: list[str] | None = field(default_factory=lambda: ["*"])
+    path_hash_prefixes: list[str] | None = field(default_factory=list)
+
+
+@dataclass
+class TestTarget:
```

Comment:
does this name not make pytest warn?

Reply:
```
PytestCollectionWarning: cannot collect test class 'TestTarget' because it has a __init__ constructor (from: tuf_conformance/test_updater_delegation_graphs.py)
```